src/vizier/services/query_broker/script_runner/cloud_source.go

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright 2018- The Pixie Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package scriptrunner
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/gofrs/uuid"
+	"github.com/gogo/protobuf/proto"
+	"github.com/gogo/protobuf/types"
+	"github.com/nats-io/nats.go"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc/metadata"
+
+	"px.dev/pixie/src/shared/cvmsgspb"
+	"px.dev/pixie/src/shared/scripts"
+	svcutils "px.dev/pixie/src/shared/services/utils"
+	"px.dev/pixie/src/vizier/services/metadata/metadatapb"
+)
+
+// CloudSource is a Source that pulls cron scripts form the cloud
```

Comment:
Nit: Typo, `form` -> `from`. Also as per the styleguide, we like to add periods at the end of these comments.